README.md

```diff
@@ -83,6 +83,8 @@ If you are going to be working with any libraries, clone them under `.libs` and
 
 Finally, deploy everything to the device by executing `npm run push` from the repository root.
 
+During the bootstrap phase a default user contract with username and password equal to jellyfish is inserted. When you are logged in with the user, grab the access token and you can start interacting with Jellyfish with the [client SDK](https://github.com/product-os/jellyfish-client-sdk), for instance.
```

Comment:
I think it's fair to mention the `jellyfish`/`jellyfish` default user, but I think it would be more useful to show how a developer can log in through the SDK or API using this user. An SDK snippet to be executed in a browser console for example. I personally use curl and JavaScript scripts that log in and get a session ID directly by executing a `action-create-session` action through the `/api/v2/action` endpoint. What do you think?